test/unit/sequence_base_test.rb

```diff
@@ -123,12 +123,12 @@ def setup
     end
 
     it 'returns value from period' do
-      { start: 'now', end: 'later' }.each do |key, value|
+      { start: '2020-05-26', end: '2020-05-26' }.each do |key, value|
         element = FHIR::Period.new(key => value)
         expected_value = if key == :start
-                           'gt' + value
+                           'gt2020-05-25T00:00:00+00:00'
```

Comment:
This tests are not correct. 1) you don't need to covert that to dateTime, server shall understand gt2020-05-25 2) gt2020-05-25 equals to gt2020-05-25T23:59:59.0000 (means excluding any time in that date)

Reply:
Does this matter? All the value needs to do is make sure is that the resource we are searching by is included in the search. In this way, gt2020-05-25 is the same as gt2020-05-25T00:00:00+00:00. I can change it to gt2020-05-25, but then we aren't testing if they can accept datetime. Should I maybe just use datetime when the original value is datetime and date when the original value is a date?

Reply:
Yes. It doesn't matter since Inferno only need to check if the resource returned.